bindings/core/src/message_interface_old/account_method.rs

```diff
@@ -48,6 +48,8 @@ pub enum AccountMethod {
     BuildAccountOutput {
         // If not provided, minimum storage deposit will be used
         amount: Option<String>,
+        #[serde(default, with = "string")]
```

Comment:
I don't know yet if mana will be mandatory to build an output, if it is, we may forget to remove this default. I'd maybe drop a temporary TODO until we know more about mana.